src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -67,4 +67,14 @@ public CommandResult execute() {
         }
     }
 
+    /**
+     * AddCommand mutates the actual data of the file
+     *
+     * @return true
+     */
```

Comment:
This is not needed as the header comment from the overridden method class still applies here as well.